examples/jsm/controls/TrackballControls.d.ts

```diff
@@ -22,6 +22,7 @@ export class TrackballControls extends EventDispatcher {
 	minDistance: number;
 	maxDistance: number;
 	keys: number[];
+	mouseButtons: { LEFT: MOUSE; MIDDLE: MOUSE; RIGHT: MOUSE };
```

Comment:
Isn't it necessary to import the enum `MOUSE`?

Reply:
Good call. I already had mouse imported for my test case. I'll add it to the file.